app/assets/stylesheets/hyrax/_accessibility.scss

```diff
@@ -0,0 +1,28 @@
+// This stylesheet is a place to apply styling changes for improving the
+// accessibility of Hyrax vanilla
+
+// Testing: WAVE evaluation tool: http://wave.webaim.org/
+// Chrome has a WAVE extension that you can toggle on/off for testing pages
+
+// Bootsrap upto v4.0 has poor styling for color-contrast that causes the WAVE
+// tool to throw contrast errors
+// Read more: https://getbootstrap.com/docs/4.0/getting-started/accessibility/#color-contrast
+// These changes meet WCAG AA and AAA compliance standard
+// * Darkens the text for active breadcrumb on every page. Meets WCAG AA and AAA
+// * Darkens the text for disabled pagination buttons on every page.
+.breadcrumb>.active,
+.pagination > .disabled > a {
+  color: #4c4c4c;
+}
+
+// NOTE: This isn't an ideal fix. We're using select2-rails v3.5
+// for styling dropdowns. This can be fixed by updating to select2-rails
+// v4.0.3 but it throws an uncaught error and fails to
+// display the dropdown at all.
+// TODO: I opened an issue that we should follow up on later
+// https://github.com/argerim/select2-rails/issues/176
+// Darkens the 'Search for a user' text in dropdown on 'Dashboard page'
+// Meets WCAG AA and AAA compliance standard
+.select2-default {
+  color: #4c4c4c !important;
```

Comment:
@samvera/hyrax-code-reviewers I'm trying to override another styling that's using `!important`. Is there a better way?

![screen shot 2017-11-16 at 3 53 38 pm](https://user-images.githubusercontent.com/21689371/32915291-62ec401c-cae6-11e7-8a58-7501d9b357a2.png)

Reply:
@h-parekh It gets really ugly but you can be more specific. I'm not sure if Hound will like it either. `html > body > .select2-default`

Reply:
I've had to do it in Sufia for Bootstrap coming from 3 different places. Again kinda ugly.

https://github.com/psu-stewardship/scholarsphere/blob/develop/app/assets/stylesheets/scholarsphere/typography.scss#L32

Reply:
I'd take this a recomendation not a dictate. I'd much rather you use `!important`

Reply:
In the interest of making the best use of time, I'm thinking that the usage of `!important` is acceptable for now. Especially because this is a temp fix until the select2-rails gem gets upgraded.